src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/GetWorkitemMetadata.sql

```diff
@@ -50,7 +65,6 @@ BEGIN
 			ON wi.WorkitemKey = eqt.SopInstanceKey1
 			AND wi.PartitionKey = eqt.PartitionKey
     WHERE
-	    wi.PartitionKey = @partitionKey
-	    AND wi.WorkitemUid = @workitemUid
+	    wi.WorkitemKey = @workitemKey
```

Comment:
Ideally we dont need this change right?

Reply:
Did that, because we already have WorkitemKey. So why not use that, instead of doing partitionkey + workitemuid look up again. But then, this wouldn't make much of a difference anyway as both will use the same clustered index.